colab/accounts/tests/test_utils_validators.py

```diff
@@ -0,0 +1,13 @@
+from django.test import TestCase
+
+from ..utils.validators import validate_social_account
+
+
+class TestValidators(TestCase):
+
+    def test_validate_social_account_with_fake_account(self):
```

Comment:
The test is making a request to twitter directly. Therefore, if no internet connection is available, or some user creates an account named 'john-fake' or delete the account 'john', the test will fail. A mock can be applied on the request to fix this issue.